README.adoc

```diff
@@ -7,6 +7,10 @@ https://coveralls.io/github/CS2103JAN2018-W13-B1/main?branch=master[image:https:
 https://www.codacy.com/app/CS2103-W13-B1-helper/main?utm_source=github.com&utm_medium=referral&utm_content=CS2103JAN2018-W13-B1/main&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/bd21fc56cd1f4c968e246a55d23ed930[Codacy Badge]]
 https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
 
+ContactSails is an address book application meant exclusively for salespersons from small businesses with customers from various social media platforms.
+
+In addition to aggregating customer details from these platforms, it can also be used for keeping track of orders and sending relevant promotions to customers.
```

Comment:
Maybe we can phrase it as "keeping track of orders from customers and sending relevant promotions to them."?

Reply:
Noted, will update